### DISCUSSION TITLE
Disable queries of enhanced introspection

### DISCUSSION THREAD

**Discussion by naotox** [Netflix/dgs-framework#1706] [Q&A]
As specified here https://www.apollographql.com/docs/federation/subgraph-spec/, enhanced introspection allows querying _service and _entities by default. I know we can set dgs.graphql.introspection.enabled to false, but it doesn't seem to apply to these two enhanced introspections.
Is there a similar flag to disable federation given that we are not using any subgraph features? I can't seem to find it.
Thanks!

**Comment by srinivasankavitha** [comment]
We don't have that as a configuration option at the moment.
